client/js/core/protocol.js

```diff
@@ -6,38 +6,168 @@ var Health = require("entities/model/health");
 var BioticState = require("entities/model/bioticState");
 var WorldItemState = require("entities/model/worldItemState");
 
-var Protocol = {};
+var Protocol = {
+	MSG_HANDSHAKE_REPLY:    0,
+	MSG_HANDSHAKE_ERROR:    1,
+	MSG_ENTITY_CREATE:      2,
+	MSG_ENTITY_STATE:       3,
+	MSG_ENTITY_REMOVE:      4,
+	MSG_CHUNK:              5,
+	MSG_BLOCK:              6,
+	MSG_CONTROLS_STATE:     7,
+	MSG_CHAT:               8,
+	MSG_DEBUG_RAY:          9,
+	MSG_NTP_SYNC:          10,
+	MSG_INVENTORY_STATE:   11,
+	MSG_INVENTORY_MOVE:    12,
+	MSG_SCOREBOARD_ADD:    13,
+	MSG_SCOREBOARD_SET:    14,
+	MSG_SCOREBOARD_REMOVE: 15
```

Comment:
What happens if this list gets out of sync with the server?

Could we meta-code it? (i.e. look at the server list to generate the client file, or something)